### ISSUE TITLE
Speed up inference

### ISSUE THREAD

**Issue by MarcoDalFarra** [PavlosMelissinos/enet-keras#14]
I'm back again. I have some pretty decent results on the Camvid dataset now thanks to your help. I have a question for you, that you might be able to anwer. I'm not able to reproduce the fast inference time. In the article they state that:

"For inference we merge batch normalization and dropout layers into the convolutional
filters, to speed up all networks."

Do you know where I can find any related litterature on how to do so or perhabs you know how they do it?

**Comment by PavlosMelissinos** [comment]
I don't know, sorry. That's definitely a new layer that combines backend operations so you're more likely to find something like that as a tensorflow implementation, otherwise you could ask the original authors to post their predict network.

**Comment by MarcoDalFarra** [comment]
I should have checked the repo for the original implementation. Someone asked this exact question a couple of days ago :) Thanks for the reply nonetheless :)

**Comment by mrgloom** [comment]
Looks it's done like this:
https://github.com/ZFTurbo/Keras-inference-time-optimizer
